packages/Graph/Actions/User/GetUserProfile.cs

```diff
@@ -30,14 +30,26 @@ public class GetUserProfile : BaseMsGraphCustomAction<User>
         [JsonProperty("UserId")]
         public StringExpression UserId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the fields to select from the Graph API.
+        /// </summary>
+        [JsonProperty("FieldsToSelect")]
+        public StringExpression FieldsToSelect { get; set; }
```

Comment:
The Graph API documentation refers to these as properties - I would rename to SelectProperties or similar to match the existing nomenclature.